content/en/graphing/using_graphs/_index.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Getting started with graphs
+kind: documentation
+description: Visualize your data to gain insight
+further_reading:
+- link: "https://learn.datadoghq.com/course/view.php?id=8"
+  tag: “Learning Center”
+  text: “Building Better Dashboards”
+---
+
+## The graphing editor
+
+There are two ways to interact with the Graphing Editor: using the GUI (the default method) and writing JSON (the more advanced method). This page covers using the GUI. To learn more about using JSON, see [Graphing Primer using JSON][1].
+
+On each graph, a pencil icon opens the graph editor:
+
+{{< img src="graphing/index/references-graphing-overview.png" alt="Graphing Overview" responsive="true" style="width:75%;" >}}
+
+The graphing editor has the following tabs:
+
+* **Share**: Embed the graph on any external web page.
+* **JSON**: The more flexible editor which requires knowledge of the graph definition language.
+* **Edit**: The default, GUI tab for graphing options.
+
+When you first open the graph editor, you are on the **Edit** tab. Here you can use the UI to choose most settings. Here is an example:
+
+{{< img src="graphing/index/references-graphing-edit-window-with-y.png" alt="Graphing Edit Tab" responsive="true" style="width:75%;" >}}
+
+Configuring a graph is a multi-step process:
+
+1. [Choose the metric to graph](#choose-the-metric-to-graph)
+2. [Select the visualization](#select-your-visualization)
+3. [Filter](#filter)
+4. [Aggregate and Rollup](#aggregate-and-rollup)
+5. [Apply additional functions](#advanced-graphing)
+6. [Title the graph](#create-a-title)
+
+## Choose the metric to graph
+
+When you create a graph, you probably have a metric in mind that you want to show. You can select that in the first dropdown under step #2, **Graph your data**. If you don't know which metric to use, you might want to start with the [Metrics Explorer][2] or a [Notebook][3]. You can also see a list of metrics in the [Metrics Summary][4].
+
+The Metrics Explorer allows you to play around with different graph settings in a more ad-hoc way. The Metrics Summary shows the type and default unit for a metric.
+
+## Select your visualization
+
+Once you have a metric in mind to display in your graph, select your visualization. Check the [list of all visualizations (widgets)][5].
```

Comment:
```suggestion
Once you select a metric to display in your graph, choose your visualization. Check the [list of all visualizations (widgets)][5].
```
This is a little backwards, but it can be done either way.